pushers/processor.go

```diff
@@ -164,12 +183,34 @@ func handlePlatformEvent(e *lambda.LambdaEvent, requestID string) bool {
 	return false
 }
 
-func process(e *lambda.LambdaEvent, cloudObj *cloud, tags map[string]string) ([]byte, error) {
+func process(e *lambda.LambdaEvent, cloudObj *cloud, tags map[string]string, requestDurations *requestDurations) ([]byte, error) {
 	eventType := e.EventType
 	timestamp := e.EventTime
 	record := e.Record
 
 	switch eventType {
+	case lambda.PlatformStart:
+		var err error
+		requestDurations.Start, err = time.Parse(time.RFC3339Nano, timestamp)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse platform.start event: %v", e)
+		}
+		if _, ok := record.(map[string]interface{}); !ok {
+			return nil, fmt.Errorf("failed to parse platform.start event: %v", e)
+		}
+		tags[lambda.StepTag] = lambda.StartStep
+		edLog := &edLog{
+			common: common{
+				Faas:       faasObj,
+				Cloud:      cloudObj,
+				LogType:    eventType,
+				LambdaTags: tags,
+				Timestamp:  timestamp,
+			},
+			Message: fmt.Sprintf("START RequestID: %s", faasObj.RequestID),
+		}
+		delete(tags, lambda.StepTag)
```

Comment:
map is a reference type, deleting this also deletes it from LambdaTags,
also the same issue below with endstep

Reply:
That's why I have deleted it actually. I just wanted to send `Step: START` and `Step: END` for only start and end logs. Otherwise all of the logs will have `Step: START` in them.

Reply:
yes but you are deleting it before marshalling to string.

Reply:
Sorry I have missed that. Thanks